modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -1714,6 +1721,25 @@ private BlogsGroupServiceConfiguration _getBlogsGroupServiceConfiguration(
 				BlogsGroupServiceConfiguration.class.getName()));
 	}
 
+	private Set<Long> _getCurrentAndAncestorSiteAndDepotGroupIds(long groupId) {
+		Set<Long> groupIds;
+
+		try {
+			groupIds = SetUtil.fromArray(
+				_siteConnectedGroupGroupProvider.
+					getCurrentAndAncestorSiteAndDepotGroupIds(groupId));
```

Comment:
Instead of saving the value in the `groupIds` variable, can you simply return the value and remove the var?